Display Phone's Index

Frappe811 opened the pull request.

Frappe811 committed:
Create index page

Frappe811 committed:
Fixed Index's Views Errorr.

Frappe811 committed:
Merge branch 'master' into index

Conflicts:
	app/assets/stylesheets/application.scss
	app/views/layouts/application.html.erb

Frappe811 committed:
Create Display Phone's Index

Frappe811 committed:
Finish Pagination.

Frappe811 committed:
Merge branch 'master' into index

Conflicts:
	app/assets/stylesheets/application.scss
	app/views/layouts/application.html.erb

ghost commented:
<img width="1440" alt="screen shot 2016-11-23 at 1 54 46 pm" src="https://cloud.githubusercontent.com/assets/23193980/20553150/79b727c0-b184-11e6-8f20-db84f9e7c011.png">

- Những điện thoại đầu tiên nên cách ra
- Những trang đầu bị trống 2 hình ở dòng cuối. Hãy để mỗi trang 6 hoặc 12 hình.
- Tên điện thoại to lên, đậm thêm chút.
- Chỉnh format cho giá tiền: 2000000 -> 2.000.000đ chẳng hạn.
- Làm pagination giống như design.

Frappe811 committed:
Finish Pagination
Nhung thay doi:
Những điện thoại đầu tiên cách ra
Mỗi trang 12 hình.
Tên điện thoại to lên, đậm thêm.
Chỉnh format cho giá tiền: 2000000.00d -> $2.000.000.
Làm pagination giống như design.

Frappe811 committed:
Coding convention

somename12321 commented on app/assets/stylesheets/phones.scss:
```diff
@@ -0,0 +1,116 @@
+    }
+
+    @media(max-width: 768px) {
+      .first, .prev, .next, .last {
```
Block này bị trống nè, bỏ đi a

Frappe811 committed:
Delete empty block
